### DISCUSSION TITLE
Is it possible to persist the "bezier cables" setting?

### DISCUSSION THREAD

**Discussion by jamshark70** [sebshader/pdnext#19] [Q&A] [answered]
Just tried this, twice to be sure:

1. Open pdnext.
2. Set "Bezier cables" = on.
3. Save all preferences.
4. Quit.
5. Open pdnext.
6. "Bezier cables" has reverted to off.

Because (I think) both pd-vanilla and pdnext are using ~/.pdsettings, I can imagine that there might be a conflict if pdnext writes a new preference into this file that vanilla doesn't understand.

At the same time, I would like a way to use Bezier cables without having to go to the menu every time. If it isn't possible to implement this in preferences, then a script or hack would be fine too.

**Comment by sebshader** [comment, marked as answer]
@jamshark70 well since both pd and pd-next use the same settings, I don't save anything extra to it. There is no persistent curve cords in the preferences themselves, but you can save a specific value for it by putting
```set ::curve_cords 1``` (to turn them on)
into a *-plugin.tcl file in the pd path.

I'll document this on the release page too.

**Comment by jamshark70** [comment]
This works for me. Thank you!
